### PR TITLE
update operator for blocking issue 37142

### PR DESCRIPTION
**Pull request by bluzarraga**

### CODE REVIEW

File: deploy/crds/operator.ibm.com_v1alpha1_helmapi_cr.yaml [hunk 1]
```diff
@@ -10,7 +10,8 @@ spec:
       enabled: true
     image:
       pullPolicy: IfNotPresent
-      repository: quay.io/opencloudio/icp-audit-service
+      imageRegistry: quay.io/cloudio
```
Comment (cheewaio) [inline deploy/crds/operator.ibm.com_v1alpha1_helmapi_cr.yaml]: imageRegistry should be quay.io/opencloudio

**Review by cheewaio — CHANGES_REQUESTED** [review]
Please fix imageRegistry value, it should be quay.io/opencloudio

File: deploy/crds/operator.ibm.com_v1alpha1_helmapi_cr.yaml (hunk 1 above)
Comment (bluzarraga) [reply deploy/crds/operator.ibm.com_v1alpha1_helmapi_cr.yaml]: whoops left that out

**Pull request merged**